server/src/instant/scripts/daily_metrics.clj

```diff
@@ -0,0 +1,127 @@
+(ns instant.scripts.daily-metrics
+  "Job to ping discord with daily active metrics. We also use this job to
+  populate the daily_app_transactions table with new transactions."
+  (:require
+   [instant.jdbc.aurora :as aurora]
+   [instant.util.date :as date]
+   [clojure.tools.logging :as log]
+   [instant.discord :as discord]
+   [chime.core :as chime-core]
+   [instant.flags :refer [get-emails]]
+   [instant.config :as config]
+   [instant.jdbc.sql :as sql]
+   [instant.grab :as grab])
+  (:import
+   (java.time Period LocalDate)))
+
+(defn excluded-emails []
+  (let [{:keys [test team friend]} (get-emails)]
+    (vec (concat test team friend))))
+
+(defn get-daily-actives
+  "Returns the number of active devs and apps for a day"
+  ([date-str]
+   (get-daily-actives (aurora/conn-pool) date-str))
+  ([conn date-str]
+   (sql/select-one conn
+                   ["SELECT
+                  dat.date as date_start,
+                  COUNT(dat.count) AS total_transactions,
+                  COUNT(DISTINCT u.id) AS distinct_users,
+                  COUNT(DISTINCT a.id) AS distinct_apps
+                FROM daily_app_transactions dat
+                JOIN apps a ON dat.app_id = a.id
+                JOIN instant_users u ON a.creator_id = u.id
+                WHERE dat.is_active
+                  AND dat.date = DATE(?)
+                  AND u.email NOT IN (SELECT unnest(?::text[]))
+                GROUP BY 1
+                ORDER BY 1"
+                    date-str
+                    (with-meta (excluded-emails) {:pgtype "text[]"})])))
+
+(defn send-discord!
+  "Ping the discord channel with the metrics for a specific date"
+  [stats date-str]
+  (let [{:keys [distinct_users distinct_apps]} stats
+        message (str "🎯 Daily metrics for " date-str
+                     ": Active Devs: **" distinct_users
+                     "**, Active Apps: **" distinct_apps
+                     "**")]
+    (discord/send! config/discord-metrics-channel-id message)))
```

Comment:
You might just want to dump this in the `team` channel? It's only 1 message a day.

Reply:
Updated!